packages/graphql-mocks/src/highlight/operation/filter.ts

```diff
@@ -2,6 +2,7 @@ import { Reference } from '../types';
 import { isEqual } from '../utils/is-equal';
 
 export function filter(source: Reference[], update: Reference[]): Reference[] {
+  // TODO: Fix filter to not use `isEqual` but something faster
```

Comment:
All usages of `isEqual` in `highlight` should be re-written to something quicker.

Reply:
Done in #274